NEWS.md

```diff
@@ -6,6 +6,7 @@
 * Optimized file upload functionality
 * Used shared data-import-utils
 * Renamed endpoints
+* Added Spring DI support
```

Comment:
this will not go to the 1.2.0 release, so put this update in the new iteration section